slick/src/main/scala/slick/jdbc/JdbcTypesComponent.scala

```diff
@@ -101,7 +102,15 @@ trait JdbcTypesComponent extends RelationalTypesComponent { self: JdbcProfile =>
     val byteArrayJdbcType = new ByteArrayJdbcType
     val charJdbcType = new CharJdbcType
     val clobJdbcType = new ClobJdbcType
+    @deprecated(message = "java.util.Date is deprecated, use java.time.LocalDateTime instead.", since = "3.2.0")
```

Comment:
We should remove the deprecated message, because it's `java.sql.Date` instead of `java.util.Date`.